README.md

```diff
@@ -15,27 +27,58 @@ Variables without a default value and not marked optional *MUST* be defined for
 | LAB_RESULTS_MESH_OUTBOUND_QUEUE_NAME | lab_results_mesh_outbound | The name of the outbound (to MESH) message queue
 | LAB_RESULTS_MESH_INBOUND_QUEUE_NAME  | lab_results_mesh_inbound  | The name of the inbound (from MESH) message queue
 | LAB_RESULTS_GP_OUTBOUND_QUEUE_NAME   | lab_results_gp_outbound   | The name of the outbound (to GP System) message queue
+| LAB_RESULTS_AMQP_BROKERS             | amqp://localhost:5672     | A comma-separated list of URLs to AMQP brokers (*)
+| LAB_RESULTS_AMQP_USERNAME            |                           | (Optional) username for the AMQP server
+| LAB_RESULTS_AMQP_PASSWORD            |                           | (Optional) password for the AMQP server
+| LAB_RESULTS_AMQP_MAX_REDELIVERIES    | 3                         | The number of times a message will be retried to be delivered to consumer. After exhausting all retries, it will be put on DLQ.<queue_name> dead-letter queue. The default JMS configuration is (-1) disabled.
+
+(*) Active/Standby: The first broker in the list is always used unless there is an error, in which case the other URLs will be used. At least one URL is required.
+
+### Mongodb Configuration Options
```

Comment:
`MongoDB`.

Sorry to be a pain, but can we update the display name version of MongoDB in these docs? Not a fan of "Mongodb" or "mongodb"

Reply:
No problem - updated this